compiler/src/typed/env.re

```diff
@@ -2245,15 +2246,10 @@ let crc_of_unit = filename => {
 
 let imports = () => {
   let imported_units = StringSet.elements(imported_units^);
-  let resolved_units =
-    List.map(
-      unit => Module_resolution.locate_unit_object_file(unit),
-      imported_units,
-    );
-  List.map2(
-    (unit, resolved_unit) => (unit, crc_of_unit(resolved_unit)),
+
+  List.map(
+    (unit) => (unit, crc_of_unit(unit)),
```

Comment:
Root cause appeared to be that `crc_of_unit` which calls `find_pers_struct` operates as if the filename passed in was to the `.gr` file rather than the `.gro` file